src/routes/docs/widget/tutorials/+page.md

```diff
@@ -7,3 +11,49 @@ Below are tutorials for popular Web frameworks/libraries, like React, Angular an
 ### Angular with Angular CLI
 
 ### Modal with Vue and Vite
+
+### "Magic links" and journey continuation
```

Comment:
I'm not totally sold on placing this doc here, but just wanted to put it somewhere for now.